core/components/atoms/_chip/index.tsx

```diff
@@ -15,10 +15,11 @@ export interface GenericChipProps extends BaseProps {
   onClose?: () => void;
   onClick?: () => void;
   name: Name;
+  children?: React.ReactNode;
```

Comment:
Please remove this prop and use the `label` prop. Introducing new prop for the same property may be confusing to integrate.